defaults/api_ents_test.go

```diff
@@ -0,0 +1,31 @@
+package defaults
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/docker/libentitlement/secprofile"
+	"github.com/docker/libentitlement/testutils"
+	"github.com/stretchr/testify/require"
+)
+
+func TestApiEntitlementEnforce(t *testing.T) {
+	entitlementID := APIEntAllowID
+	require.NotNil(t, DefaultEntitlements[entitlementID])
+
+	ociProfile := secprofile.NewOCIProfile(testutils.TestSpec(), "test-profile")
+	apiEnt := DefaultEntitlements[entitlementID]
+
+	_, err := apiEnt.Enforce(ociProfile)
+	require.Equal(t, fmt.Errorf("OCI profile's APIAccess field nil"), err)
+
+	//	require.NoError(t, err, "Failed to enforce while testing entitlement %s", entitlementID)
```

Comment:
You can probably remove all these comments as we won't test the OCI specs content in this entitlement